client/my-sites/plans-features-main/index.jsx

```diff
@@ -165,7 +173,9 @@ export class PlansFeaturesMain extends Component {
 		}
 
 		let plans;
-		if ( group === GROUP_JETPACK ) {
+		if ( plansProp && plansProp.length ) {
+			plans = plansProp.map( type => findPlansKeys( { group, term, type } )[ 0 ] );
```

Comment:
Now that I look at it further, the `plans` prop should probably just take the plans directly (i.e. `PLAN_BUSINESS`) rather than the type (i.e. `TYPE_BUSINESS`).

Reply:
At second thought, maybe not, as the term and group settings could also modify the plans. Maybe this prop should be renamed `planTypes`.

Reply:
Agreed w/ second thought. Renamed.